app/src/components/form/FormProgressBar.tsx

```
@@ -0,0 +1,255 @@
+    if (onNavigate) {
+      await onNavigate(step.path);
+    } else {
+      // Default navigation
+      navigate({ to: step.path as any });
+    }
+  };
+
+  const getStepStyles = (state: StepState, underlyingState: StepState) => {
+    switch (state) {
+      case "current":
+        let fillColor = "bg-gray-300"; // default
```

Biome (2.4.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)